ibis/backends/base/__init__.py

```diff
@@ -42,6 +42,16 @@
 }
 
 
+_SQLALCHEMY_TO_SQLGLOT_DIALECT = {
```

Comment:
Can we add a comment here to clarify that all of the other dialects use the same identifier for sqlalchemy and sqlglot?

Reply:
Yup

Reply:
Done!